browser/resources/settings/email_aliases_page/email_aliases.tsx

```diff
@@ -0,0 +1,227 @@
+import { MainEmailEntryForm } from './content/email_aliases_signin_page'
+import { AliasList } from './content/email_aliases_list'
+import styled, { StyleSheetManager } from 'styled-components'
+
+const MainEmailTextContainer = styled(Col)`
+  justify-content: center;
+  cursor: default;
+  user-select: none;
+`
+
+const MainEmailDescription = styled.div`
+  font: ${font.default.regular};
+`
+
+const MainEmail = styled.div`
+  font: ${font.large.semibold};
+  padding-bottom: ${spacing.s};
+`
+
+const AccountRow = styled(Row)`
+  justify-content: space-between;
+  align-items: center;
+  padding: ${spacing.l} 0px ${spacing.l};
+`
+
+const ManageAccountButton = styled.button`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font: ${font.default.regular};
+  align-items: center;
+  color: ${color.text.secondary};
+  text-decoration: none;
+  background: none;
+  border: none;
+  border-radius: 0.5em;
+  padding: 0.5em;
+  & > span {
+    margin: 0.25em;
+  }
+  cursor: pointer;
+  &:hover {
+    background-color: var(--leo-color-desktopbrowser-toolbar-button-hover);
+  }
+  &:active {
+    background-color: var(--leo-color-desktopbrowser-toolbar-button-active);
+  }
+`
+
+const PageCol = styled(Col)`
+  padding: ${spacing.l};
+`
+
+const Introduction = () =>
+  <Card>
+    <h2>{getLocale('emailAliasesShortDescription')}</h2>
+    <div>{getLocale('emailAliasesDescription')} &emsp;
+      {/* TODO: Link to the email aliases support page */}
+      <SecureLink href="https://support.brave.com" target='_blank'>
+        {getLocale('emailAliasesLearnMore')}
+      </SecureLink>
+    </div>
+  </Card>
+
+const MainEmailDisplay = ({ email, onLogout }: { email: string, onLogout: () => void }) =>
+  <Card>
+    <AccountRow>
+      <Row>
+        <BraveIconCircle name='brave-icon-release-color' />
+        <MainEmailTextContainer>
+          <MainEmail>{email === '' ? getLocale('emailAliasesConnectingToBraveAccount') : email}</MainEmail>
+          <MainEmailDescription>{getLocale('emailAliasesBraveAccount')}</MainEmailDescription>
+        </MainEmailTextContainer>
+      </Row>
+      <ManageAccountButton
+        title={getLocale('emailAliasesSignOutTitle')}
+        onClick={(e) => { e.preventDefault(); onLogout() }}>
+        <Icon name="outside" />
+        <span>{getLocale('emailAliasesSignOut')}</span>
+      </ManageAccountButton>
+    </AccountRow>
+  </Card>
+
+
+const MainView = ({
+  viewState, mainEmail, onLogout, aliasesState, setViewState,
+  mappingService, onListChange
+}: {
+  viewState: ViewState,
+  mainEmail: string,
+  onLogout: () => void,
+  aliasesState: Alias[],
+  setViewState: (viewState: ViewState) => void,
+  mappingService: MappingService,
+  onListChange: () => void
+}) =>
+  (viewState.mode === 'Startup' ?
+    <Row style={{ margin: '1em', flexGrow: 1, justifyContent: 'center', alignItems: 'center' }}><Icon name='loading-spinner' />
```

Comment:
nit: avoid inline styles
nit: newline
```suggestion
    <Row style={{ margin: '1em', flexGrow: 1, justifyContent: 'center', alignItems: 'center' }}>
        <Icon name='loading-spinner' />
```

Reply:
fixed